urllib3/poolmanager.py

```diff
@@ -112,6 +113,8 @@ class PoolManager(RequestMethods):
 
     def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+        if "transport_security_manager" not in connection_pool_kw:
+            connection_pool_kw["transport_security_manager"] = TransportSecurityManager()
```

Comment:
To key a pool off `transport_security_manager`, it needs to be added to the `HTTPPoolKey` object. That should be sufficient, though you'll want to add a test like those in `test_poolmanager.py` to confirm that the keying is working appropriately.

What this means, essentially, is that if the user *changes* the TSM on the pool manager, all existing pools will no longer be usable for new connections.